src/SourceGenerators/Uno.UI.SourceGenerators/Telemetry/TelemetryClient.cs

```diff
@@ -120,12 +122,24 @@ private void InitializeTelemetry()
 		{
 			try
 			{
+				_storageDirectoryPath = Path.Combine(Path.GetTempPath(), ".uno", "telemetry");
+
+				// Store the settings on in the user profile for linux
+				if (RuntimeEnvironment.OperatingSystemPlatform == Platform.Linux)
+				{
+					_settingsStorageDirectoryPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".uno", "telemetry");
+				}
+				else
+				{
+					_settingsStorageDirectoryPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Uno Platform", "telemetry");
```

Comment:
This will work on an unpackaged WinUI project?

Reply:
We're not in the context of winui for this code.